### PR TITLE
AWSEMFExporter - Adding SummaryDataType, Remove MinMax from Histogram

### PR DESCRIPTION
**Pull request by shaochengwang**
**Description:** 
Add SummaryDataType 
Remove Min Max from Histogram data type

**Link to tracking Issue:** 
https://github.com/open-telemetry/opentelemetry-collector-contrib/issues/1510

**Testing:** 
unit tests
manual end-to-end test

**Documentation:**
Add SummaryDataType since it's now supported
Remove Min Max from Histogram data type since the histogram buckets doesn't expose the Min and Max.

### CODE REVIEW

**Review by anuraaga — APPROVED** [review]

File: exporter/awsemfexporter/metric_translator.go [hunk 1]
```diff
@@ -82,6 +82,7 @@ type CWMetricStats struct {
 // 	- pdata.DoubleDataPointSlice
 // 	- pdata.IntHistogramDataPointSlice
 // 	- pdata.DoubleHistogramDataPointSlice
+//  - pdata.DoubleSummaryDataPointSlice
```
Comment (bogdandrutu) [inline exporter/awsemfexporter/metric_translator.go:85]: Align with previous comment?

File: exporter/awsemfexporter/metric_translator.go [hunk 2]
```diff
@@ -92,6 +93,7 @@ type DataPoints interface {
 // 	- pdata.DoubleDataPoint
 // 	- pdata.IntHistogramDataPoint
 // 	- pdata.DoubleHistogramDataPoint
+//  - pdata.DoubleSummaryDataPointSlice
```
Comment (bogdandrutu) [inline exporter/awsemfexporter/metric_translator.go:96]: ditto

File: exporter/awsemfexporter/metric_translator.go (hunk 1 above)
Comment (shaochengwang) [reply exporter/awsemfexporter/metric_translator.go:85]: Sure. Didn't see this in the IDE. I will replace all tabs with spaces :)

**Review by mxiamxia — APPROVED** [review]
Thanks!

**Comment by shaochengwang** [comment]
Sorry, I accidentally touched several commits. I have reopened a new PR to clean this up. https://github.com/open-telemetry/opentelemetry-collector-contrib/pull/1584
Closing this one.

**Pull request closed without merging**